src/user/user.module.ts

```diff
@@ -3,10 +3,11 @@ import { UserController } from './user.controller';
 import { UserService } from './user.service';
 import { PrismaService } from '../prisma/prisma.service';
 import { UserRepository } from './user.repository';
+import { PendingUserDeletion } from './pending-user-cron';
 
 @Module({
   controllers: [UserController],
-  providers: [UserService, UserRepository, PrismaService],
+  providers: [UserService, UserRepository, PrismaService, PendingUserDeletion],
```

Comment:
the name does not say what this class is exactly. It should be e.g. `PendingUserDeletionCronService`